src/components/TextInput/BaseTextInput.js

```diff
@@ -2,10 +2,9 @@ import _ from 'underscore';
 import React, {Component} from 'react';
 import {
     Animated, View, TouchableWithoutFeedback, Pressable, AppState, Keyboard,
-    // eslint-disable-next-line no-restricted-imports
-    TextInput as RNTextInput,
 } from 'react-native';
 import Str from 'expensify-common/lib/str';
+import RNTextInput from '../BaseTextInput';
```

Comment:
```suggestion
import BaseTextInput from '../BaseTextInput';
```

Please follow the same name as the Component name.

Reply:
the class here is BaseTextInput.js

so we have to rename one of them @parasharrajat  

which one should be the base

Reply:
I think it shoud be the base not just for leftAligined .  
if we need to add more default props to all text input we will use this component.

Reply:
I think it shoud be the base not just for leftAligined .  
if we need to add more default props to all text input we will use this component.

Reply:
maybe  `BaseInput`  is better name

Reply:
@parasharrajat  maybe `BaseInput` is better name

Reply:
I don't have preferences. Maybe Marc can help us with a good name.

Reply:
I don't have preferences. Maybe Marc can help us with a good name.